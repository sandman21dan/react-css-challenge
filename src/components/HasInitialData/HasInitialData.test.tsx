import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { HasInitialData } from '.';

it('renders correctly when no data', () => {
  const tree = renderer.create(<HasInitialData hasInitialData={false}/>).toJSON();

  expect(tree).toMatchSnapshot();
});

it('renders correctly when there is data', () => {
  const tree = renderer.create(<HasInitialData hasInitialData={true}/>).toJSON();

  expect(tree).toMatchSnapshot();
});
