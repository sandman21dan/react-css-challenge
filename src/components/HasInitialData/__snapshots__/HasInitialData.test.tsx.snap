// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when no data 1`] = `
<p>
  Loading...
</p>
`;

exports[`renders correctly when there is data 1`] = `
<p>
  Got initial data
</p>
`;
